lib/screens/v2/candidate_generator/widgets/reconstructed_alert.ex

```diff
@@ -74,4 +78,18 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlert do
           false
       end
   end
+
+  defp get_stations(alert) do
+    %{alert: alert}
+    |> BaseAlert.informed_entities()
+    |> Enum.map(fn %{stop: stop_id} -> stop_id end)
+    |> Enum.filter(&String.starts_with?(&1, "place-"))
+    |> Enum.uniq()
+    |> Enum.map(
+      case &Stop.fetch_stop_name(&1) do
+        :error -> ""
```

Comment:
You might want to use `Enum.flat_map` instead of `Enum.map`, and have the callback function return an empty list in case of `:error`.

Currently, if any `Stop.fetch_stop_name` call fails, you'll get a list that looks like `["Stop Name 1", "", "Stop Name 3"]`. When that gets passed to `Util.format_name_list_to_string`, it will produce `"Stop Name 1, , and Stop Name 3"`.

If you use `Enum.flat_map`, this error-handling clause can return an empty list, which will remove the element completely. You'll get `["Stop Name 1", "Stop Name 3"]` which formats to `"Stop Name 1 and Stop Name 3"`.